internal/pkg/api/server/buffer/dependencies/dependencies.go

```diff
@@ -130,6 +132,9 @@ func NewServerDeps(serverCtx context.Context, envs env.Provider, logger log.Pref
 		return nil, errors.New("KBC_STORAGE_API_HOST environment variable is not set")
 	}
 
+	// Get Buffer API host
+	bufferApiHost := strhelper.NormalizeHost(envs.MustGet("KBC_BUFFER_API_HOST"))
```

Comment:
Lepsie by bolo pouzit envs.Get a vratit chybu.
Kedze sa nejedna iba o nejaku vyvojarsku chybu, ale staci, ze niekto v k8s sa nenastavi env,
a lepsie bude ked sa vypise do logov nejaka zrozumitelna hlaska, ako panic so stack trace.

Reply:
Alebo `envs.Lookup` (ta vrati `found bool`).
Staci ked namiesto `MustGet` das `Get` a nizsie skontrolujes, ze hodnota nie je prazdna.

Reply:
Má to být stejně tak i pro `KBC_STORAGE_API_HOST`? Tam je taky `MustGet`

Reply:
jj, uprav to prosim aj tam